package.json

```diff
@@ -51,6 +51,7 @@
     "asap": "https://github.com/alloy/asap.git",
     "react": "^15.4.2",
     "react-relay": "https://github.com/alloy/relay/releases/download/v0.9.3/react-relay-0.9.3.tgz",
+    "react-url-query": "^1.1.1",
```

Comment:
Ah, this is where I decided to cut myself off. Started down the path of making a component to manage url state and pass it down to filter components.